src/libraries/System.Formats.Cbor/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+﻿<Project>
+  <Import Project="..\Directory.Build.props" />
+  <PropertyGroup>
+    <StrongNameKeyId>Microsoft</StrongNameKeyId>
+    <IsNETCoreApp>true</IsNETCoreApp>
```

Comment:
I think this means "Is part of the shared framework", and I don't think CBOR needs to be in the shared framework, just a standalone NuGet package.  Right?

Reply:
fixed